Phases/Assessments/Phase_1.md

```diff
@@ -0,0 +1,37 @@
+# Phase 1 Assessment
+
+The phase 1 assessment will test your ability to write basic JavaScript programs for the Node.js platform as well as your fundamental SQL knowledge.
+
+You will _not_ be assessed on your HTML, CSS, or other web-specific skills.
+
+## Skills Assessed
+
+- Programming (Beginner Level)
+- JavaScript (Beginner Level)
```

Comment:
Reading further in this document, "Beginner" also makes sense in context and relates this level of tasks to other concepts.  For example, what should `Code Editor (Beginner Level)` be to relate this level of knowledge to the JavaScript category?